lib/components/map/default-map.js

```diff
@@ -141,6 +141,13 @@ class DefaultMap extends Component {
 
     return (
       <MapContainer>
+        {/* Add the mapbox wordmark if set in config. The implementing
+          application must include CSS that properly displays the wordmark. See
+          mapbox website for example CSS:
+          https://docs.mapbox.com/help/how-mapbox-works/attribution/#other-mapping-frameworks */}
+        {mapConfig.addMapboxWordmark &&
+          <a href='http://mapbox.com/about/maps' class='mapbox-wordmark' target='_blank'>Mapbox</a>
+        }
```

Comment:
I don't love the idea of adding an additional config property like this. It's just additional (unnecessary) configuration. What if we just check the active layer URL and add the wordmark if it comes from `mapbox.com`.